tests/test_routes.py

```diff
@@ -162,7 +180,7 @@ def test_not_found(self):
 
     def test_method_not_allowed(self):
         """Test if using an unsupported HTTP method returns a 405 Method Not Allowed"""
-        response = self.client.put("/recommendations/1")
+        response = self.client.post("/recommendations/1")
```

Comment:
Maybe a better idea to also test "update without an id," reference: https://github.com/nyu-devops/lab-flask-tdd/blob/91420f2498faa8461ba2f63796f2a7b13aef1071/tests/test_routes.py#L240-L243

Reply:
Agree. I have changed it to post method without an id.